SQL/paradise_schema.sql

```diff
@@ -119,10 +120,12 @@ CREATE TABLE `death` (
   `laname` text NOT NULL COMMENT 'Last attacker name',
   `lakey` text NOT NULL COMMENT 'Last attacker key',
   `gender` text NOT NULL,
+  `species` text NOT NULL,
   `bruteloss` int(11) NOT NULL,
   `brainloss` int(11) NOT NULL,
   `fireloss` int(11) NOT NULL,
   `oxyloss` int(11) NOT NULL,
+  `map` varchar(64) NOT NULL DEFAULT 'NSS Cyberiad',
```

Comment:
Were defaulting to cyberiad because thats what we use for 999/1000 rounds.

Reply:
I didn't even know the other maps were acknowledged server-side save from test merges